assets/webpack.config.js

```diff
@@ -55,7 +55,7 @@ module.exports = (env, options) => {
       }),
       new MiniCssExtractPlugin({ filename: "../css/app.css" }),
       new MonacoWebpackPlugin({
-        languages: ["markdown", "elixir", "xml", "json"],
+        languages: ["markdown", "elixir", "xml", "json", "sql"],
```

Comment:
I wonder if in the future we should download those dynamically (and finally move away from webpack) but there is no urgency right now.

Reply:
This list specifies which language files are copied to the bundle and we cherry-pick mostly because we keep assets in memory for the sake of Escript. The other thing is that the plugin handles some boilerplate code and configuration, so it's quite convenient.

But yeah, it would make sense to consider migration from webpack, just need to consider stuff like code-splitting and some other features we use.